api/src/opentrons/protocol_engine/state/geometry.py

```diff
@@ -106,8 +106,8 @@ def get_labware_parent_nominal_position(self, labware_id: str) -> Point:
         elif labware_data.location == OFF_DECK_LOCATION:
             # Labware is off-deck
             raise errors.LabwareNotOnDeckError(
-                f"Labware {labware_id} does not have a parent associated with it"
-                f" since it is no longer on the deck."
+                f"Cannot access labware {labware_id} since it is no longer on the deck. "
```

Comment:
Thanks for this better error message.

One clarification:

```suggestion
                f"Cannot access labware {labware_id} since it is not on the deck. "
```

Since the labware might not have been on the deck to begin with, if it was loaded off-deck.